packages/codegen-ui-react/lib/__tests__/__snapshots__/studio-ui-codegen-react-forms.test.ts.snap

```diff
@@ -5920,56 +5920,47 @@ export default function TagUpdateForm(props) {
     ...rest
   } = props;
   const initialValues = {
-    label: undefined,
-    Posts: [],
+    name: undefined,
+    Team: undefined,
   };
-  const [label, setLabel] = React.useState(initialValues.label);
-  const [Posts, setPosts] = React.useState(initialValues.Posts);
+  const [name, setName] = React.useState(initialValues.name);
+  const [Team, setTeam] = React.useState(initialValues.Team);
   const [errors, setErrors] = React.useState({});
   const resetStateValues = () => {
-    const cleanValues = tagRecord
-      ? { ...initialValues, ...tagRecord, Posts: linkedPosts }
+    const cleanValues = memberRecord
+      ? { ...initialValues, ...memberRecord }
```

Comment:
Will the reset be a bit of problem? `cleanValues.Team` will be a promise then if simply spread `memberRecord`?

```
    const cleanValues = memberRecord
      ? { ...initialValues, ...memberRecord, Team }
```

and will have one more dependency

```
React.useEffect(resetStateValues, [memberRecord, Team]);
```